.github/workflows/ci.yml

```diff
@@ -51,6 +51,8 @@ jobs:
         refs/heads/master
         refs/heads/develop
       ', github.ref)
+    needs:
+      - test
```

Comment:
That is quite a temptation, I agree, but it's not going to work as you expect.
The chart-testing does not run on the main branch, only on pull request. The chart-releaser the opposite, only on the main, not on pull requests.
So in fact, with this change, the releaser will never run.

Reply:
Ah, well dang. This is where I learn about the differences between CircleCI (which I used a _lot_) and GitHub Actions.

I think accomplishing this, somehow, is going to need to be a requirement going forward though. Publishing before knowing if the tests pass is... not great. I guess I'll have to dig into that. Thanks for the hint, I'll pull that out for now.